### PR TITLE
Mosseljongen

### PR DESCRIPTION
**Pull request by TayfunAygun**
Pull request voor level

### CODE REVIEW

File: Assets/ProBuilder Data/Default Material Palette.asset.meta [hunk 1]
```diff
@@ -1,5 +1,9 @@
 fileFormatVersion: 2
+<<<<<<< HEAD
+guid: ebd539e2ed5f2e84bb4a095437caa348
+=======
 guid: 862aba347cff0874293c26db96bb9c6a
+>>>>>>> master
 NativeFormatImporter:
   externalObjects: {}
   mainObjectFileID: 0
```
Comment (mvanwoensel) [inline Assets/ProBuilder Data/Default Material Palette.asset.meta:9]: Not sure what you are changing here, as you are changing stuff in the shared assets

File: Assets/FPS/Prefabs/Pickups/Loot_Lightball.prefab [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+      propertyPath: m_LocalEulerAnglesHint.y
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 6409866517097999930, guid: 4d1efaa062e333f4cbc4c3564702d135,
+        type: 3}
+      propertyPath: m_LocalEulerAnglesHint.z
+      value: 0
+      objectReference: {fileID: 0}
+    m_RemovedComponents: []
+  m_SourcePrefab: {fileID: 100100000, guid: 4d1efaa062e333f4cbc4c3564702d135, type: 3}
```
Comment (mvanwoensel) [inline Assets/FPS/Prefabs/Pickups/Loot_Lightball.prefab]: Put this is your own asset folder for now, as it's something you made on your own / we didnt discuss to add it for everybody (and you know in this way you get credit for making it i guess?)

File: ProjectSettings/Packages/com.unity.probuilder/Settings.json [hunk 3]
```diff
@@ -37,181 +37,6 @@
...
-            },
-            {
-                "type": "UnityEditor.ProBuilder.Actions.DetachFaces+DetachSetting, Unity.ProBuilder.Editor, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null",
-                "key": "DetachFaces.target",
-                "value": "{\"m_Value\":0}"
-            },
-            {
-                "type": "UnityEditor.ProBuilder.Actions.DuplicateFaces+DuplicateFaceSetting, Unity.ProBuilder.Editor, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null",
-                "key": "DuplicateFaces.target",
-                "value": "{\"m_Value\":0}"
```
Comment (mvanwoensel) [inline ProjectSettings/Packages/com.unity.probuilder/Settings.json:214]: could you explain what you are doing here?

**Review by mvanwoensel — CHANGES_REQUESTED** [review]
Look at the comments i made

**Review by mvanwoensel — APPROVED** [review]
lgtm

File: Assets/FPS/Prefabs/Pickups/Loot_Lightball.prefab (hunk 2 above)
Comment (TayfunAygun) [reply Assets/FPS/Prefabs/Pickups/Loot_Lightball.prefab]: Done

File: ProjectSettings/Packages/com.unity.probuilder/Settings.json (hunk 3 above)
Comment (TayfunAygun) [reply ProjectSettings/Packages/com.unity.probuilder/Settings.json:214]: No idea what is happening here.

File: Assets/ProBuilder Data/Default Material Palette.asset.meta (hunk 1 above)
Comment (TayfunAygun) [reply Assets/ProBuilder Data/Default Material Palette.asset.meta:9]: I have overwritten my .meta files with the .meta files from the master. Maro told me what to do.

**Pull request merged**